seismic_site/map_app/models.py

```diff
@@ -26,46 +22,52 @@ class SeismicCategoryChoice(Enum):
 
 
 class Building(models.Model):
-    id_general = models.AutoField(primary_key=True)
-    clasa_categorie = models.CharField(max_length=50, db_index=True)
-    nr_pmb = models.IntegerField(null=True)
+    general_id = models.AutoField(primary_key=True)
+
+    risk_category = models.CharField(max_length=50, db_index=True)
+    registration_number = models.IntegerField(null=True)
+    examination_year = models.IntegerField(null=True)
+    certified_expert = models.CharField(max_length=100, null=True)
+    observations = models.CharField(max_length=1000, null=True)
+
     lat = models.FloatField(null=True)
-    long = models.FloatField(null=True)
-    loc = models.CharField(max_length=60)
-    adresa = models.CharField(max_length=250, null=True)
-    nr_postal = models.CharField(max_length=250)
-    sector = models.CharField(max_length=20)
-    nr_sector = models.IntegerField(null=True)
-    an_construire = models.IntegerField(null=True)
-    regim_inaltime = models.CharField(max_length=50)
-    nr_apart = models.IntegerField(null=True)
-    arie_desfasurata = models.FloatField(null=True)
-    an_expertiza = models.IntegerField(null=True)
-    expert_atestat = models.CharField(max_length=100)
-    obs = models.CharField(max_length=1000)
-    numar_cadastral = models.IntegerField(null=True)
-    nr_carte_funciara = models.CharField(max_length=50)
-    actualizare_pmb = models.DateField(null=True, blank=True)
-    editare_admin = models.DateField(null=True, blank=True)
+    lon = models.FloatField(null=True)
```

Comment:
`lng` is commonly used

Reply:
never heard of this one until now, but if other SW makers use it, then OK; I'll try to make the changes tonight

Reply:
It's generally not good to edit the migrations but I've made an exception this time :)

Reply:
It's a newly added migration so it's only bad for your env  😄